src/main/java/software/sigma/internship/entity/user/Person.java

```diff
@@ -0,0 +1,22 @@
+package software.sigma.internship.entity.user;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
```

Comment:
please, don't use * in imports. Also, configure your idea no to replace imports to *